package.json

```diff
@@ -22,7 +22,7 @@
     "build": "wvr build --clean"
   },
   "dependencies": {
-    "@wvr/core": "2.2.4",
+    "@wvr/core": "https://github.com/TAMULib/Weaver-UI-Core.git#modal-keyboard-escape-fix",
```

Comment:
Be sure to revert/change this once weaver-ui-core is updated to the supporting weaver-ui-core release.